src/qililab/platform/platform.py

```diff
@@ -574,13 +574,34 @@ def execute_qprogram(  # pylint: disable=too-many-locals
         calibration: Calibration | None = None,
         debug: bool = False,
     ) -> QProgramResults:
-        """Execute a QProgram using the platform instruments.
+        """Execute a :class:`.QProgram` using the platform instruments.
+
+        |
+
+        **The execution is done in the following steps:**
+
+        1. Compile the QProgram.
+        2. Run the compiled QProgram.
+        3. Acquire the results.
+
+        |
+
+        **The execution can be done for (buses associated to) two different type of clusters:**
+
+        - For ``Qblox`` modules, the compilation is done using the :class:`.QbloxCompiler`. Which compiles the :class:`.QProgram` into``Q1ASM`` for multiple sequencers based on each bus, uploads and executes the sequences, acquires the results and resets the settings by desynching each bus.
```

Comment:
`resets the settings`: This is not true. The settings of the instruments are not touched. The only setting that is changed is the syncing between buses that are involved in the qprogram. But this is only during execution.

Reply:
What do you think about the new change:
```
- For ``Qblox`` ... and executes the sequences (desynching buses if necessary), and acquires the results.
```

Reply:
I can also just delete de parenthesis if you prefer 🙌

Reply:
If we don't mention the syncing part before execution, then we shouldn't mention the desyncing afterwards, since it is confusing. So without the parentheses is better I think.